lib/rules/declaration-empty-line-before/__tests__/index.js

```diff
@@ -32,46 +33,49 @@ testRule(rule, {
 
   reject: [ {
     code: "a { top: 15px; }",
+    fixed: "a {\n\n top: 15px; }",
```

Comment:
Problem here is that we're going to have to decide how to indent or not indent. I think keeping the space is not right, though: that space should be replaced by the newlines, not added to.